### ISSUE TITLE
As a SO I would like to be able to embed my current newsletter as a block on a page

### ISSUE THREAD

**Issue by ekellagher** [CuBoulder/express#2003]
It feels cumbersome to update a site with the current newsletter when you want to have the current newsletter in the site navigation. I would like to have the newsletter content on a page and then have the newsletter list block in the right sidebar so visitors may read the current newsletter and see a list of the all the newsletters and maybe a button to be put on an Email List. The newsletter does not have an Edit layout option so maybe the newsletter could be pulled into a current newsletter block.

here is a site who has requested this option:
https://www.colorado.edu/controller/newsletter/mar-2018-edition

Below is a mockup

![newsletter example](https://user-images.githubusercontent.com/16137919/37123852-f17df6b6-2222-11e8-9a08-40f6bbfb0b6b.png)
